posthog/queries/funnels/funnel_trends.py

```diff
@@ -162,11 +167,12 @@ def _summarize_data(self, results):
         breakdown_clause = self._get_breakdown_prop()
 
         summary = []
+
         for period_row in results:
             serialized_result = {
                 "timestamp": period_row[0],
-                "reached_from_step_count": period_row[1],
-                "reached_to_step_count": period_row[2],
+                "reached_from_step_count": correct_result_for_sampling(period_row[1], self._filter.sampling_factor),
+                "reached_to_step_count": correct_result_for_sampling(period_row[2], self._filter.sampling_factor),
```

Comment:
Might make sense to include the correction in the query itself. Query classes can piggy back off `self._filter.sampling_factor` and add the adjustment if necessary. Then none of these count format functions need to consider this

Reply:
Just saw the issue and consideration for this that happened already!

Reply:
Fair, I suppose if there's no real performance gains for doing it query-side, python side makes sense

Reply:
Coming back to this discussion to say that while I feel strongly about doing more on the Python side, I do think it's always a case by case thing so will be considering how to best approach sampling going forward.

I thought it'd make things messy to do math and handle zeros and the like on the query side, further polluting already complicated queries. But I might give a query a shot and see how it turns out. It definitely ensures we don't miss a spot later down where we need to account for sampling